Library/Formula/libdecnumber.rb

```diff
@@ -0,0 +1,41 @@
+class Libdecnumber < Formula
+  desc "ANSI C implementation of General Decimal Arithmetic"
+  homepage "http://speleotrove.com/decimal/#decNumber"
+  url "http://download.icu-project.org/files/decNumber/decNumber-icu-368.zip"
+  version "3.68"
+  sha256 "14ec2cf30b58758493a7661b78b80abfb281652b61a425b85cda83173518fe25"
+  resource "cmakelists" do
```

Comment:
I don't think I understand what you are trying to achieve here. I wrote the
CMakeLists. That's a script in a language made specifically to build stuff.
Yet you suggest I write the same thing in Ruby? I would likely get that
wrong so i really don't see the advantage.

I am not sure why the developer did not provide a build file. It is likely
because this is a very low level library mostly used by compilers so the
couple of compilers that use it, incorporate the sources in their code and
use their own build system to build it.
On Dec 4, 2015 9:11 AM, "Dominyk Tiller" <notifications@github.com> wrote:

> In Library/Formula/libdecnumber.rb
> <https://github.com/Homebrew/homebrew/pull/46647#discussion_r46657724>:
>
> > @@ -0,0 +1,41 @@
> > +class Libdecnumber < Formula
> > +  desc "ANSI C implementation of General Decimal Arithmetic"
> > +  homepage "http://speleotrove.com/decimal/#decNumber"
> > +  url "http://download.icu-project.org/files/decNumber/decNumber-icu-368.zip"
> > +  version "3.68"
> > +  sha256 "14ec2cf30b58758493a7661b78b80abfb281652b61a425b85cda83173518fe25"
> > +  resource "cmakelists" do
>
> Manually, if necessary. If upstream are making it this difficult to build
> it, it's hard not to wonder if they intend it to be packaged by anyone in *this
> manner* rather than embedded in other sources.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Homebrew/homebrew/pull/46647/files#r46657724>.
>